contracts/test/tx_allow_list.ts

```diff
@@ -68,4 +68,18 @@ describe("ExampleTxAllowList", function () {
   test("should not let allowed to revoke admin", "step_onlyAdminCanRevoke")
 
   test("should let admin to revoke allowed", "step_adminCanRevoke")
+
+  test("should let manager to revoke allowed", "step_managerCanRevoke")
+
+  test("should let manager to add allowed", "step_managerCanAllow")
+
+  test("should not let manager to add admin", "step_managerCannotGrantAdmin")
+
+  test("should not let manager to revoke admin", "step_managerCannotRevokeAdmin")
+
+  test("should not let manager to revoke manager", "step_managerCannotRevokeManager")
+
+  test("should not let manager to add manager", "step_managerCannotGrantManager")
+
+  test("should let manager to deploy", "step_managerCanDeploy")
```

Comment:
```suggestion
  test("should let manager to transact", "step_managerCanDeploy")
```

Reply:
replied here: https://github.com/ava-labs/subnet-evm/pull/801#discussion_r1309297349

If we want to send txs instead of deployments. it should be in it's own PR; as it would require a change to test contract.